packages/rlc-common/src/buildObjectTypes.ts

```diff
@@ -46,6 +49,40 @@ export function buildObjectInterfaces(
   return objectInterfaces;
 }
 
+export function buildObjectAliases(
```

Comment:
Do we still need to build aliases expect the paged cases?

Reply:
Not sure I get your point, Paged has already been added into the core-paging.

Reply:
I mean the `buildObjectAliases` is to get all alias definition.

Currently we have one case to generate alias, that is paged.

Reply:
no, for polymorphism, we also use alias.